.github/workflows/core.yml

```diff
@@ -342,8 +353,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: [ 3.7, 3.8 ]
+        python: [ 3.9 ]
```

Comment:
why not modify to  [ 3.7, 3.8, 3.9 ] ?

Reply:
Because Flink 1.15 does not support Python 3.9

Reply:
I think we don't need to test all combinations of Java and Python.

`all_java_version * default_python_version` plus `default_java_version * all_python_version_except_default_one` is sufficient.

Reply:
> I think we don't need to test all combinations of Java and Python.
> 
> `all_java_version * default_python_version` plus `default_java_version * all_python_version_except_default_one` is sufficient.

Thank you very much for explaining.